examples_tests/37.EXRSplit/main.cpp

```diff
@@ -27,7 +27,7 @@ int main()
 	auto am = device->getAssetManager();
 
 	asset::IAssetLoader::SAssetLoadParams lp;
-	auto image_bundle = am->getAsset("../../media/daily_pt_1.exr", lp);
+	auto image_bundle = am->getAsset("../../media/OpenEXR/daily_pt_1.exr", lp);
```

Comment:
still native file dialog or command like arg to give us the filename (I want this to be a tool I actually want to use rather than a pointless example)